src/modules/utils.js

```diff
@@ -258,4 +266,3 @@ function createUUIDPart(length) {
     }
```

Comment:
We currently bundle all our dependencies for VSCode and would like to keep our dependency list to a minimum. Whenever we take a new dependency we need to take it to our legal department to verify the license. However, if we find more cases where we need a new library, we are definitely open to it.